backend/routes/chat/brainful_chat.py

```diff
@@ -32,6 +32,7 @@ def get_answer_generator(
         user_openai_api_key,
         streaming,
         prompt_id,
+        user_id,
     ):
```

Comment:
We should use the user_id and brain_id to be attributes of the class BrainlessChat and BrainfullChat